### PR TITLE
stubbed cases for PAT

### PR DESCRIPTION
**Pull request by akhil-jha**
Stubbed cases for new feature Personal Access Token. (API/CLI/UI)
issues: 600

### CODE REVIEW

**Review by latran — APPROVED** [review]
ACK

**Review by synkd — APPROVED** [review]
ACK

File: tests/foreman/api/test_user.py [hunk 1]
```diff
@@ -884,3 +884,68 @@ def test_positive_access_entities_from_ipa_org_admin(self, create_ldap):
...
+        :steps:
+            1. Set the expired time to 1 minute from the current time.
+            2. Wait 1 minute
+            3. Try using the token with any end point.
+
+        :expectedresults: Authentication error
+
+        :CaseLevel: System
+        """
+        pass
```
Comment (devendra104) [inline tests/foreman/api/test_user.py:951]: Remove the `pass` from all the test cases

File: tests/foreman/ui/test_user.py [hunk 2]
```diff
@@ -334,3 +334,67 @@ def test_positive_create_product_with_limited_user_permission(
...
+
+    :id: 8e46907f-cf56-4e77-8496-8d8935b333d8
+
+    :steps:
+        1. Set the expired time to 1 minute from the current time.
+        2. Wait 1 minute
+        3. Try using the token with any end point.
+
+    :expectedresults: Authentication error
+
```
Comment (devendra104) [inline tests/foreman/ui/test_user.py:397]: Case Importance is missing in all the test cases

**Review by devendra104 — APPROVED** [review]
ACK

**Pull request merged**